tcpclient/connection.go

```diff
@@ -19,6 +19,7 @@ type connectionMetrics struct {
 	// packets lost, retransmissions and other metrics could come
 }
 
+// Types of connection status
 const (
 	ConnectionNotInitiated ConnectionStatus = 0
```

Comment:
do you mind improving something on this? This sequence can take advantage of https://github.com/golang/go/wiki/Iota

Reply:
👍